dotCMS/src/main/java/com/dotcms/rest/api/v1/container/ContainerResource.java

```diff
@@ -214,6 +214,9 @@ public final Response containerContent(@Context final HttpServletRequest req,
 
         try {
 
+            req.getSession().setAttribute(com.liferay.portal.util.WebKeys.USER_ID, user.getUserId());
+            PrincipalThreadLocal.setName(user.getUserId());
```

Comment:
Should the webResource do this work?  Also, not sure about setting the session, shouldn't the auth take care of this?

Reply:
Yeah, we discussed about the session yesterday and Jonathan is going to check if the authentication process sets that session attribute, I would say it does but we are going to check.
And yes, we can see if there is a better place the ThreadLocal code.